cdap-common/src/main/java/co/cask/cdap/common/conf/Constants.java

```diff
@@ -512,22 +512,19 @@
     public static final String SERVER_ADDRESS = "metrics.query.bind.address";
     public static final String SERVER_PORT = "metrics.query.bind.port";
 
-    public static final String KAFKA_TOPIC_PREFIX = "metrics.kafka.topic.prefix";
```

Comment:
But after upgrade, will no longer be used anywhere in CDAP code. I thought as long as I kept the metrics topic prefix value "metrics" consistent, there will be no problem for Kafka to fetch and process remaining messages